packages/simplebar-vue/__snapshots__/index.test.js.snap

```diff
@@ -1,49 +1,110 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`simplebar renders with options 1`] = `
-<div data-simplebar="" data-simplebar-auto-hide="false">
-  <div class="simplebar-wrapper">
-    <div class="simplebar-height-auto-observer-wrapper">
-      <div class="simplebar-height-auto-observer"></div>
+<div
+  data-simplebar=""
+  data-simplebar-auto-hide="false"
+>
```

Comment:
hey, do you know why these changes happened? It seems like a prettifier passed here.
In theory we have a precommit hook with Prettier so idk what happened. if you could revert this before merge.

Reply:
It was because I used `wrapper.element` when making snapshots.
https://vue-test-utils.vuejs.org/guides/testing-single-file-components-with-jest.html#snapshot-testing
I added `jest-serializer-vue` now so snapshots are the same as before.

Reply:
Hey actually I thought this was the source, I didn't realise I was commenting on a snapshot file!
That's totally fine to have these changes then haha